### PR TITLE
rtsm linux: add minimal linux configuration with submodule

### PR DESCRIPTION
**Pull request by in1004kyu**
Guest linux included submodule "linux-rtsm"
Add filesystem file "fs.cpio", device tree binary file "host-a15.dtb"
Add usage at README.md, how test linux guest on RTSM with bare-metal guest.

README.md
...
Testing minimal linux on RTSM fastmodels using git submodule
Download linux-rtsm
$ git submodule init
$ git submodule update
Initial setup for linux
$ cd linux-rtsm
$ git checkout 7d1f9aeff1ee4a20b1aeb377dd0f579fe9647619 -b 3.8
$ git apply ../patch/linux-fastmodels-config-add-minimal-linux-config.patch
$ cp ../linuxguest/fs.cpio ./
$ cp ../linuxguest/host-a15.dtb ./
Building linux-arndale
$ make ARCH=arm vexpress_minhw_defconfig
$ CROSS_COMPILE=arm-linux-gnueabi- ARCH=arm make -j8
$ cat host-a15.dtb >> arch/arm/boot/zImage
Loading khypervisor to RTSM fastmodels
$ cp arch/arm/boot/zImage ../monitor/zImage
$ cd ../monitor
$ make LINUX=y
Run and Debug on DS-5 or FastModels MaxView + RTSM VE Cortex A15-A7
$ maxview &
Under monitor/ directory 
$ ./run_rtsm.sh
...

### CODE REVIEW

File: monitor/loadlinux.h [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        struct atag_cmdline cmdline;
+    } u;
+};
+
+void setup_core_tag( void* address, long pagesize );
+void setup_revision_tag( void );
+void setup_cmdline_tag( const char* line );
+void setup_mem_tag( uint32_t start, uint32_t len );
+void setup_end_tag( void );
+void setup_tags( uint32_t *src );
```
Comment (simonkim) [inline monitor/loadlinux.h]: 1. Move all the private macros and functions into loadlinux.c 
2. Rename API function: 
   - setup_tags() -> loadlinux_setup_tags( uint32_t *src );

File: monitor/loadlinux.c [hunk 2]
```diff
@@ -0,0 +1,71 @@
+#include "loadlinux.h"
+#include "string.h"
+#include "print.h"
+
+
+static struct atag *params; /* used to point at the current tag */
```
Comment (simonkim) [inline monitor/loadlinux.c]: - params is a module internal (private) variable not accessible from outside
- Therefore, suggest to use '_' prefix convention
  - _params;

File: linux-rtsm [hunk 3]
```diff
@@ -0,0 +1 @@
+Subproject commit af91706d5ddecb4a9858cca9e90d463037cfd498
```
Comment (simonkim) [inline linux-rtsm]: make it point to 7d1f9aeff1ee4a20b1aeb377dd0f579fe9647619 instead of af91706d5ddecb4a9858cca9e90d463037cfd498

**Comment by in1004kyu** [comment]
@simonkim apply comment.

**Comment by simonkim** [comment]
@in1004kyu why me? where? about what?

**Pull request merged**